test/api/visualization/clusters_controller_test.rb

```diff
@@ -75,6 +76,17 @@ class ClustersControllerTest < ActionDispatch::IntegrationTest
     assert_equal [1, 4], bar_data['x']
   end
 
+  test 'show should handle numeric clusters' do
+    sign_in_and_update @user
+    execute_http_request(:get, api_v1_study_cluster_path(@basic_study, 'clusterA.txt', {annotation_name: 'intensity', annotation_scope: 'cluster', annotation_type: 'numeric'}))
+    assert_equal 'Reds', json['data'][0]['marker']['colorscale']
+    @basic_study.update!({default_options: {color_profile: 'Portland'}})
+    # since we're going to send a duplicate request, clear the API cache so we can get fresh results
+    Rails.cache.clear
```

Comment:
Non-blocking: rather than the brute-force `Rails.cache.clear`, you can call `CacheRemovalJob.new(@basic_study.accession).perform` to clear the visualization caches for only this study.